juniper_official/Solutions/VPLS-view/vpls.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
478
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/VPLS-view/vpls.yml
  7:1       error    syntax error: could not find expected ':'